src/js/components/Signature.jsx

```diff
@@ -14,8 +14,19 @@ const showReceipt = (code, timestamp, receipt) => {
   )
 }
 
-const showWarning = (code) => {
-  if(code > 8) return null
+const showWarning = (props) => {
+  if(!props.error && props.status.code > 8) return null
+
+  if(props.error) {
+    return (
```

Comment:
Why not use the error component here? If it's for the specific message maybe another `prop` would help? My initial take is that it would be nice to have this done in one place.